integration/externalbuilders/golang/bin/build

```diff
@@ -27,7 +27,7 @@ if [ -f "$SOURCE/src/go.mod" ]; then
     cd "$SOURCE/src"
     go build -v -mod=readonly -o "$OUTPUT/chaincode" "$GO_PACKAGE_PATH"
 else
-    GO111MODULE=off go build -v  -o "$OUTPUT/chaincode" "$GO_PACKAGE_PATH"
+    GOPATH="$SOURCE" GO111MODULE=off go build -v  -o "$OUTPUT/chaincode" "$GO_PACKAGE_PATH"
```

Comment:
So before this gopath was completely unset, and was the default set depending on go version, and the fact that it built was because everything was already checked out in the gopath. I think this makes sense.